extension/persistence/eclipselink/build.gradle.kts

```diff
@@ -65,6 +65,10 @@ dependencies {
   compileOnly("com.fasterxml.jackson.core:jackson-annotations")
   compileOnly("com.fasterxml.jackson.core:jackson-core")
 
+  // JDBC Drivers
+  runtimeOnly(libs.h2)
+  runtimeOnly(libs.postgresql)
```

Comment:
If you're going back to in-memory by default, these changes aren't needed either

Reply:
I think it is valuable to end users that Polaris's normal build includes usable JDBC drivers. I do not think it is viable to force users to recompile to include PostgreSQL.